packages/entity-list/src/components/Table/Table.js

```diff
@@ -200,7 +221,9 @@ Table.propTypes = {
     reverseRelationName: PropTypes.string
   }),
   showLink: PropTypes.bool,
-  linkFactory: PropTypes.objectOf(PropTypes.func)
+  linkFactory: PropTypes.objectOf(PropTypes.func),
+  changePosition: PropTypes.func.isRequired,
+  positions: PropTypes.objectOf(PropTypes.number)
```

Comment:
`positions` actually required? (i guess there will be no errors if it's undefined, but would it make sense?)

Reply:
there will be an error in the useDnD hook.

Reply:
in which case? if `positions` is undefined? then I suppose we should append the `isRequired`?